.eslintrc

```diff
@@ -160,6 +160,11 @@
               {
                 "group": ["lodash", "lodash/*"],
                 "message": "lodash is a commonjs module, which breaks webpack esm optimizations."
+              },
+              {
+                "group": ["react"],
```

Comment:
I guess this would still not stop people from using `React.useLayoutEffect`? But hopefully that would get caught in PR as incorrect style anyway